BasicSample/Assets/Interaction/Scripts/HapticsSupport.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR;
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    public class HapticsSupport : MonoBehaviour
```

Comment:
It might be good to add a `<summary>` tag to describe what this class is doing/showing

Reply:
Is it better to name this class "ApplyHapticsSample"?  it will show up in the unity editor better.

Reply:
@yl-msft since this class is part of the interaction sample and not a sample on its own, should i still call it ApplyHapticsSample? Instead is it better to name it just ApplyHaptics?

Reply:
i agree with you,  the namespace has "sample", so  "ApplyHaptics" is enough.

Reply:
Done